core/sql/regress/tools/runregr_udr.ksh

```diff
@@ -851,9 +851,9 @@ if [ $LINUX -ne 0 ]; then
 fi
 
```

Comment:
Should be fine Hans.  Just did not want to see hpt4jdbc.jar being referenced anywhere incorrectly as our tyep4 jar file is now called jdbcT4.jar